### DISCUSSION TITLE
Intel Free86 -> Nvidia FOSS gimme a cookie New kernel

### DISCUSSION THREAD

**Discussion by TomTravis** [dnschneid/crouton#4844] [Q&A] [answered]
it not working yet

majic software lol

any suggestion are welcome

**Comment by MeneerJansen** [comment]
?

**Comment by TomTravis** [comment]
no cookie it is not working yet convert back to intel for free86

Install (nvidia) did lots of work  however maybe i keep working on this

**Comment by TomTravis** [comment]
kali likes Nvidia
![Screenshot from 2022-11-02 20-07-57](https://user-images.githubusercontent.com/46386261/199644492-c10a1a68-d409-4254-94ca-cf9b98f70acf.png)
cookie time

**Reply by dnschneid** [reply]
I don't think that screenshot shows your system using the nvidia driver, unfortunately.  You can still eat the cookie though.

**Reply by TomTravis** [reply]
try it you tell me what it is not dummy

even you know the config call to load dummy the controls (nvidia) work
 
see the pull request and watch the kali debootstrap install the kernel is patch for nvidia kms

**Comment by TomTravis** [comment, marked as answer]
move to chroot systemd project
